src/Microsoft.Health.Dicom.SqlServer/Features/Schema/Migrations/4.sql

```diff
@@ -1528,6 +1558,49 @@ BEGIN
 END
 GO
 
+/***************************************************************************************/
+-- STORED PROCEDURE
+--     GetExtendedQueryTagErrors
+--
+-- DESCRIPTION
+--     Gets the extended query tag errors by tag path.
+--
+-- PARAMETERS
+--     @tagPath
+--         * The TagPath for the extended query tag for which we retrieve error(s).
+--
+-- RETURN VALUE
+--     The tag error fields and the corresponding instance UIDs.
+/***************************************************************************************/
+CREATE OR ALTER PROCEDURE dbo.GetExtendedQueryTagErrors (@tagPath VARCHAR(64))
+AS
+    SET NOCOUNT     ON
+    SET XACT_ABORT  ON
+    BEGIN TRANSACTION
+
+        DECLARE @tagKey INT
+        SELECT @tagKey = TagKey
```

Comment:
Do we need this check? What happens if this check does not exist. 

Eventual consistency on gets/reads are fine.

Reply:
This is so that if the client queries for an extended query tag error that does not have an existing tag, we show an error, instead of them thinking that the tag has no error.